packages/react-resizable-panels/src/PanelResizeHandle.test.tsx

```diff
@@ -279,6 +279,18 @@ describe("PanelResizeHandle", () => {
       verifyAttribute(leftElement, "data-resize-handle-active", null);
       verifyAttribute(rightElement, "data-resize-handle-active", null);
     });
+
+    it("should stop dragging when the mouse goes beyond the browser size", () => {
+      const { leftElement, rightElement } = setupMockedGroup();
+      verifyAttribute(leftElement, "data-resize-handle-active", null);
+      verifyAttribute(rightElement, "data-resize-handle-active", null);
+
+      act(() => {
+        dispatchPointerEvent("pointermove", leftElement, { clientX: -1 });
+      });
+      verifyAttribute(leftElement, "data-resize-handle-state", "inactive");
+      verifyAttribute(rightElement, "data-resize-handle-state", "inactive");
+    });
```

Comment:
This test seems to give a false positive. (I can remove the change you made entirely and it will still pass.)

Reply:
FWIW I also think this fix may make the most sense in the resize handle registry (where other Window event listeners are) but I haven't thought about it _much_

Reply:
The reason the test gives a false positive is that the handle is not _down_ when the move event is fired. I will take a pass at fixing the test.

Reply:
Sorry, I forgot to leave an update. As I was looking at this the other day, I realize that I already had code in place to handle this (in the resize handler registry) and it mostly seems to work, but it doesn't work in the case of Code Sandbox (probably something specific to iframes seemingly not receiving "pointerleave" events). Might be worth focusing on that use case.

Reply:
It occurs to me that my e2e tests need to include a route that uses an iframe at the root. I'll add one.

Reply:
Interestingly, having just added an iframe– this library seems to work fine. So I'm not sure what it is about Code Sandbox that's different. 🤔

Reply:
Oh! I think I just realized what's going on by accidentally pushing a reference to `localhost` in my commit. I think the issue is when the iframe is in a different domain than the parent frame. Lemme poke at that a bit more.

Reply:
Okay, I'm able to trigger a similar behavior by using `sandbox="allow-scripts"`. I'll move forward from there and try to add a failing e2e test.

Reply:
Check out #374; I think I found a fix.